stable/prometheus-operator/ci/test-values.yaml

```diff
@@ -1395,6 +1395,14 @@ prometheus:
     ##
     retention: 10d
 
+    ## Maximum size of metrics
+    ##
+    retentionSize: ""
+
+    ## Enable compression of the write-ahead log using Snappy.
+    ##
+    walCompression: false
```

Comment:
This feature requires PrometheusOperator 0.32.0 https://github.com/coreos/prometheus-operator/releases/tag/v0.32.0

Reply:
Fixed with https://github.com/helm/charts/pull/16635/commits/391433c7b167f2d95c0346c15b74a96d023aa8bb